src/pages/edge/adobe-journey-optimizer-decisioning/tabs/api-reference.md

```diff
@@ -140,6 +140,53 @@ Optimize.getPropositions(scopes, new AdobeCallbackWithError<Map<DecisionScope, O
 });
 ```
 
+<Variant platform="android" api="get-propositions-withTimeout" repeat="6"/>
+
+#### Java
+
+#### Syntax
+
+```java
+public static void getPropositions(final List<DecisionScope> decisionScopes, final long timeoutMillis, final AdobeCallback<Map<DecisionScope, OptimizeProposition>> callback)
+```
+
+* _decisionScopes_ is a list of decision scopes for which propositions are requested.
+* _timeoutMillis_ is a duration in milliseconds specifying the maximum time to wait for the event to complete.
```

Comment:
Can we change this so that both platforms accept the timeout in the same unit: either milliseconds or seconds. Lets be consistent